test/scripts/test-webserver.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
-r2 -qq -c=h - &
+r2 -N -e http.port=9393 -qq -c=h bins/elf/arg > /dev/null 2>&1 &
 CHILD=$!
 sleep 2
-curl -s --retry 6 --retry-connrefused http://localhost:9090/ | head -n 8
```

Comment:
why commented?

Reply:
probably because that line might come back into play